app/views/hyrax/admin/workflow_roles/index.html.erb

```diff
@@ -26,15 +26,17 @@
         <h2 class="panel-title h2"><%= t('.current_roles') %></h3>
       </div>
       <div class="panel-body">
-        <table class="table table-striped">
+        <table class="table table-striped datatable">
           <thead>
+            <th><%= t('.header.display_name') %></th>
             <th><%= t('.header.user') %></th>
             <th><%= t('.header.roles') %></th>
           </thead>
           <tbody>
           <% @presenter.users.each do |user| %>
             <tr>
-              <td><%= user.user_key %></td>
+              <td data-sort="<%= user.display_name %>"><%= user.display_name %></td>
```

Comment:
Should we use `User#name` instead? https://github.com/samvera/hyrax/blob/b498d8ba181f4e76da89cae60e92f9f54675fad5/app/models/concerns/hyrax/user.rb#L115-L119